en/04_Changelogs/5.0.0.md

```diff
@@ -100,6 +126,18 @@ This is a major release and as a result there are a number of breaking API chang
 - `SilverStripe\Core\Extensible::invokeWithExtensions()` and `SilverStripe\Core\Extensible::extend()` now use the splat operator instead of having a concrete number of possible arguments.
 - The constructor for `SilverStripe\Versioned\Versioned` now explicitly only accepts mode as a single argument.
 
+### Email {#api-email}
+
+- Email was sent in CMS 4 using [SwiftMailer](https://swiftmailer.symfony.com/docs/introduction.html), which has since [gone End Of Life](https://symfony.com/blog/the-end-of-swiftmailer). In CMS 5 this has been replaced with [symfony/mailer](https://symfony.com/doc/current/mailer.html). `symfony/mailer` is the currently maintained email package from Symfony. It's a more flexible email system that allows easier integration with third-party email providers.
```

Comment:
We probably need to mention in here somewhere that `Email` is no longer a subclass of `ViewableData`, so anything people want to use in templates must be returned from `getData()`.

If the `getData()` method isn't mentioned in the main email docs, it also should be mentioned there (probably in the same area that talks about replacing the default template).

Reply:
I'll add a bullet point to mention the subclass thing. The main email docs already talks about using setData() - https://docs.silverstripe.org/en/4/developer_guides/email/#templates